src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme()
```

Comment:
@timosachsenberg Here it's initialised by using Enzyme::Enzyme(), I don't know if it's still able to make Enzyme::Enzyme() private.